.github/workflows/Deploy.yaml

```diff
@@ -90,3 +119,38 @@ jobs:
             Write-Host "::Error::Error deploying repositories. The error was $($_.Exception.Message)"
             exit 1
           }
+
+      - name: Calculate Release Notes
+        if: github.repository_owner == 'microsoft'
```

Comment:
Check for github.event.inputs.createRelease - No need to run this if we aren't creating a release